packages/client/tests/functional/optimistic-concurrency-control/tests.ts

```diff
@@ -98,4 +98,30 @@ testMatrix.setupTestSuite(({ provider }) => {
 
     expect(await prisma.resource.findFirst()).toMatchObject({ occStamp: 1 })
   })
+
+  test('update with upsert relation', async () => {
```

Comment:
Does this have a related issue we could refer to? 
```suggestion
  // Related https://github.com/prisma/prisma/issues/foo-bar?
  test('update with upsert relation', async () => {
```

Reply:
Found it https://github.com/prisma/prisma/issues/15470